code/datums/traits/negative.dm

```diff
@@ -154,7 +154,7 @@
 				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
 			if("Detective")
 				heirloom_type = /obj/item/reagent_containers/food/drinks/bottle/whiskey
-			if("Lawyer")
+			if("Lawyer", "Internal Affairs Agent")
```

Comment:
Why are you keeping lawyer here if you are gonna remove it?
And if anything, use job defines

Reply:
gone